tests/unit_tests/rigid_pendulum_poc/test_heavy_top.cpp

```diff
@@ -6,6 +6,37 @@
 
 namespace openturbine::rigid_pendulum::tests {
 
+TEST(HeavyTopProblemFromBrulsAndCardona2010PaperTest, CalculateResidualVector) {
+    auto M = MassMatrix(15., Vector(0.234375, 0.46875, 0.234375));
+    auto mass_matrix = M.GetMassMatrix();
+    auto rotation_matrix = create_matrix(
+        {{0.617251, -0.757955, 0.210962},
```

Comment:
I'm usually not enthusiastic about comments, but in this case it might be nice to put a note that references the paper or source for these hard-coded constants.